spec/cbra/graph_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Cbra::Graph do
+  describe "#generate!" do
+    let(:fixture_root) do
+      File.expand_path(
+        "../fixtures",
+        File.dirname(__FILE__)
+      )
+    end
+
+    let(:root) do
+      File.expand_path(
+        "../../../cbra",
```

Comment:
That's the gem itself. Is there a better way to the gem's root? Maybe starting it with `/` is all I need, I'll try that...